### ISSUE TITLE
Saved data in stores such as selectedAccount, connected, chainId removes on refresh?

### ISSUE THREAD

**Issue by manishshahi1** [clbrge/svelte-web3#37]
I'm facing a problem in which when I am refreshing a page, the data is lost.

**Comment by clbrge** [comment]
Yes, that's how store works. Data is not persistent when reloading. You will have to implement that yourself on your side. By example saving the account and connection type in localstorage and trying automatic connection when these infos are detected on reload.
I'm open to have plugin implementing this but not in core package.

**Comment by manishshahi1** [comment]
I will give it a try! Thanks
